Orange/distance/__init__.py

```diff
@@ -152,3 +152,29 @@ def __call__(self, e1, e2=None, axis=1, impute=False):
 
 PearsonR = PearsonDistance(absolute=False)
 PearsonRAbsolute = PearsonDistance(absolute=True)
+
+
+class Mahalanobis():
+    """Mahalanobis distance."""
+    def __init__(self, data, axis=1):
+        """
+        :param data: The dataset used for calculating covariances.
```

Comment:
You can set PyCharm to generate docstrings in preferred format in Settings > Tools > Python Integrated Tools > Docstring format.